### PR TITLE
[feat]: Adding Cart Page

### PR DESCRIPTION
**Pull request by raghav2110**
Hi , I have added cart page for ecom website. Please review my PR.

### CODE REVIEW

**Review by shivampandey0 — COMMENTED** [review]
Good to merge,
maybe `size` variables from the component library can be used here.

File: css/cartList.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+.main-wrapper h1{
+    margin: 1rem;
+    align-items: center;
+    justify-content: center;
+}
+
+.cart-wrapper{
+    display: grid;
+    grid-template-columns: 2.5fr 1.25fr;
```
Comment (ZuberDunge) [inline css/cartList.css]: you can also just use 2fr and 21fr here.

File: css/cartList.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+.order-summary-card{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    border: 1px solid var(--headerbg);
+    height: 16.45rem;
+    padding: 1rem;
+    color: #6b7280;
```
Comment (ZuberDunge) [inline css/cartList.css:77]: for colors, you can use variables here

**Review by ZuberDunge — APPROVED** [review]
Added few suggestions, Apart from that Code is Clean and readable, UI looks good and responsive. Good work.

File: css/cartList.css (hunk 1 above)
Comment (raghav2110) [reply css/cartList.css]: Okay noted this change

File: css/cartList.css (hunk 2 above)
Comment (raghav2110) [reply css/cartList.css:77]: It was just used once, so used it explicitly

**Review by shivampandey0 — APPROVED** [review]
Good to merge.

**Review by ZuberDunge — APPROVED** [review]

**Pull request merged**